.changeset/great-houses-bake.md

```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/design-system-styles': patch
+---
+
+Updated the utilities API mixin to shorten output CSS as well as removed unnecessary code regarding pseudo elements in that same mixin.
```

Comment:
Might be too technical for a changeset as users do not know about the utility API and mixin, only about the utility classes. 
```suggestion
Merged utility classes that were setting the same properties, which reduces the size of the CSS output.
```